index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html class="page" lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Word-Guess-Game</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+   
+</head>
+
+<body class="page">
+    
+<embed autoplay height="0" width="0" src="assets/music/spiritedAway.mp3"></embed>
+    
```

Comment:
The `<embed>` tag is "self-closing", so you don't need to include an ending `</embed>` tag.